client/components/embed-container/index.jsx

```diff
@@ -20,7 +21,15 @@ const embedsToLookFor = {
 	'blockquote[class^="instagram-"]': embedInstagram,
 	'blockquote[class^="twitter-"]': embedTwitter,
 	'fb\\\:post, [class^=fb-]': embedFacebook,
-	'[class^=tumblr-]': embedTumblr
+	'[class^=tumblr-]': embedTumblr,
+	'.jetpack-slideshow': embedSlideshow
+};
+
+const SLIDESHOW_URLS = {
+	CSS: 'https://s0.wp.com/wp-content/mu-plugins/shortcodes/css/slideshow-shortcode.css',
```

Comment:
we may need to cache bust these urls. They have `Expires: Mon, 29 May 2017 17:10:33 GMT` which means changes wouldn't be picked up very quickly. 

How we're going to keep these in sync is ... fun.